doc/usage/examples/kuberay/kuberay-mcad.md

```diff
@@ -4,13 +4,43 @@ This integration will help in queuing on [kuberay](https://github.com/ray-projec
 
 #### Prerequisites
 
-- kubernetes or Openshift cluster 
-- Install MCAD using instructions present under `deployment` directory
-- Make sure MCAD has clusterrole to create ray resources, please patch using configuration file present in `config` directory with name `xqueuejob-controller.yaml`
+- Kubernetes(see [KinD](https://helm.sh/docs/intro/install/)) or Openshift cluster(see [OpenShift Local](https://developers.redhat.com/products/openshift-local/overview))
+- Kubernetes client tools such as [kubectl](https://kubernetes.io/docs/tasks/tools/) or [OpenShift CLI](https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html)
+- [Helm](https://helm.sh/docs/intro/install/)
+- Install a stable MCAD release on your Kubernetes cluster using helm
+    ```bash
+    git clone https://github.com/project-codeflare/multi-cluster-app-dispatcher
```

Comment:
Should this be geared towards the operator instead of the helm charts?

Reply:
We can add  the ODH [quick start](https://github.com/opendatahub-io/distributed-workloads/blob/main/Quick-Start.md) for Openshift. 
Reason for keeping KInD is:
1. This doc is linked by Kuberay guide. The Kuberay community uses KinD in the other [integration guides](https://github.com/ray-project/kuberay#kuberay-ecosystem).
2. We don't have our operator  on the [operatorhub](https://operatorhub.io/),  installing with helm  is simple.

Reply:
Sounds good